common/lib/siodb/common/io/JsonWriter.cpp

```diff
@@ -179,7 +179,8 @@ void JsonWriter::writeRawString(const char* s, std::size_t length)
         }
 
         const char c = *p;
-        if (static_cast<unsigned char>(c) >= static_cast<unsigned char>(' ') && c != '/') {
+        if (static_cast<unsigned char>(c) >= static_cast<unsigned char>(' ') && c != '/' && c != '"'
```

Comment:
I've rechecked [JSON spec](https://www.json.org/json-en.html), and I think escaping of the forward slash can be avoided, because it falls in the proper character range and is not an excluded character. Please remove it from here (`&& c != '/'`) and below switch.